src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inversed = {};
+
+  for (const key in robot) {
+    if (inversed.hasOwnProperty(robot[key])) {
+      return null;
+    }
```

Comment:
You should use `Object.prototype.hasOwnProperty()` instead of just `.hasOwnProperty`

Reply:
Sorry, but i did not understand what do u mean.  Look at the screenshot from checklist: https://prnt.sc/LOFeMFoO9cwc

Reply:
I've read about it in the style-guide. 
![image](https://user-images.githubusercontent.com/101983641/209836680-8d2fd911-fc51-440b-aa33-a523d464c278.png)
here https://github.com/mate-academy/style-guides/blob/master/javascript.md#37-do-not-callobjectprototypemethods-directly-such-ashasownpropertypropertyisenumerable-andisprototypeof